src/components/Card/Card.tsx

```diff
@@ -1,8 +1,9 @@
 import type { ComponentProps, FC, PropsWithChildren } from 'react';
 import { twMerge } from 'tailwind-merge';
-import type { DeepPartial, FlowbiteBoolean } from '../../';
-import { useTheme } from '../../';
```

Comment:
Is it okay to remove these relativ imports for the `~/src` below? If not, I would be curious why.

Reply:
Does the `yarn build:lib` still working? If it does, no issue here...

Reply:
Hey @levino anything inside `src/` should just be relative paths `../../` as opposed to the `~`. Unfortunately when we use the absolute path it seems to cause issues for `flowbite-react` once it's actually shipped out on `npm`. At the moment we are retaining `tsconfig.json` paths solely to make writing the documentation in `.mdx` files easier (since your editor won't automatically write `import`s for you in this file format).

Reply:
Fixed.